pkg/image/generated/listers/image/internalversion/imagestreammapping.go

```diff
@@ -0,0 +1,78 @@
+// This file was automatically generated by lister-gen
```

Comment:
already done, that commit will go away once this merge https://github.com/openshift/origin/pull/15625